include/openthread/instance.h

```diff
@@ -285,6 +285,66 @@ otLogLevel otGetDynamicLogLevel(otInstance *aInstance);
 ThreadError otSetDynamicLogLevel(otInstance *aInstance, otLogLevel aLogLevel);
 
 /**
+ * This function pointer is called when the timer expires.
+ *
+ * @param[in]  aContext  A pointer to application-specific context.
+ *
+ */
+typedef void (OTCALL *Handler)(void *aContext);
```

Comment:
Maybe the name causes confuse, here I would like to define a function pointer for timer callback, similar to `otSetStateChangedCallback`. For application, it would use this macro to define a function pointer which is used to register a timer callback.